js/theme.js

```diff
@@ -261,3 +261,8 @@ if (downloadNote.length >= 1) {
 } else {
     $(".pytorch-call-to-action-links").hide();
 }
+
+$("p.caption:first").on("click", function() {
+    $(this).toggleClass("pytorch-left-nav-collapsed", "pytorch-left-nav-collapsible");
+    $(this).closest("#pytorch-left-menu").find(this).next("ul").toggle();
```

Comment:
I think something like `$(this).next("ul").toggle();` or `$("#pytorch-left-menu").find('ul:first').toggle();` might be more straightforward than `$(this).closest("#pytorch-left-menu").find(this).next("ul").toggle();`.